src/canvas.class.ts

```diff
@@ -738,21 +741,6 @@ import { FabricObject } from './shapes/object.class';
         return origin;
       },
 
-      /**
-       * @private
-       * @param {Boolean} alreadySelected true if target is already selected
-       * @param {String} corner a string representing the corner ml, mr, tl ...
-       * @param {Event} e Event object
-       * @param {fabric.Object} [target] inserted back to help overriding. Unused
-       */
-      _getActionFromCorner: function (alreadySelected, corner, e, target) {
-        if (!corner || !alreadySelected) {
-          return 'drag';
-        }
-        var control = target.controls[corner];
-        return control.getActionName(e, control, target);
-      },
```

Comment:
I don't remember the reason of this function.
It says something about overriding, and if is an utile imported directly can't be overridden.
I ll dig in the past to understand why this should or could be overridden.

Reply:
makes sense
so you can override to which control you are passing the event..?